package-lock.json

```diff
@@ -15,6 +15,7 @@
         "@actions/exec": "^1.1.1",
         "@actions/glob": "^0.4.0",
         "@actions/io": "^1.1.3",
+        "github-actions.cache-buildjet": "^0.2.0",
```

Comment:
pls revert this from the package-lock as well :-)

Reply:
Thanks, I've reverted this.